Dockerfile

```
@@ -26,7 +26,8 @@ RUN ln -snf /usr/share/zoneinfo/$TZ /etc/localtime && \
     apt-get install -y vim-tiny less && \
     ln -s /usr/bin/vim.tiny /usr/bin/vim && \
     rm -rf /var/lib/apt/lists/*
-
+RUN wget https://dl.min.io/client/mc/release/linux-amd64/mc
```

I think here is better to pin mc with specific version, just to be on the safe side if they remove some functionality.
```suggestion
RUN wget -O mc https://dl.min.io/client/mc/release/linux-amd64/archive/mc.RELEASE.2022-05-09T04-08-26Z
```